### PR TITLE
DOC-5467: Added pre-install checks info to Corda deployment steps

### PR DESCRIPTION
**Pull request by AnnaK-R3**
Preview: https://anna.preview.docs.r3.com/en/platform/corda/5.1/deploying-operating/deployment/deploying.html#pre-install-checks

### CODE REVIEW

File: content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md [hunk 1]
```diff
@@ -661,6 +662,29 @@ workers:
...
+* Kafka is up and if the credentials work.
+
+You can also run those checks manually using the Corda CLI [`preinstall`]({{< relref "../../../reference/corda-cli/preinstall.md" >}})
+command after you have deployed Corda.
+
+If you want to disable the pre-install checks from running automatically, for example, to save time when testing the deployment,
+you can do it by adding the following property to your YAML file:
+
+```yaml
+bootstrap.preinstallCheck.enabled=false
```
Comment (davidcurrie) [inline content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md:682]: in YAML this should be:
```suggestion
bootstrap:
  preinstallCheck:
    enabled: false
```

**Review by davidcurrie — CHANGES_REQUESTED** [review]

**Review by davidcurrie — APPROVED** [review]
LGTM

**Review by davidcurrie — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md [hunk 2]
```diff
@@ -661,6 +662,31 @@ workers:
       annotation-key-2: "some-value"
 ```
 
+### Pre-Install Checks
+
+When deploying Corda, pre-install checks are performed automatically to check the configuration of Corda and to confirm
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md:667]:
```suggestion
When deploying Corda, pre-install checks are performed automatically to check the configuration of Corda, to confirm
```

File: content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md [hunk 3]
```diff
@@ -661,6 +662,31 @@ workers:
...
+are provided to install a running Corda cluster. The pre-install checks verify if:
+
+* the resource limits have been assigned correctly.
+* the PostgreSQL database is up and if the credentials work.
+* Kafka is up and if the credentials work.
+
+You can also run those checks manually using the Corda CLI <a href="../../reference/corda-cli/preinstall.html">`preinstall`</a>
+command after you have deployed Corda.
+
+If you want to disable the pre-install checks from running automatically, for example, to save time when testing the deployment,
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md:678]:
```suggestion
If you want to disable the pre-install checks from running automatically (for example, to save time when testing the deployment), 
```

File: content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md [hunk 4]
```diff
@@ -661,6 +662,31 @@ workers:
...
+you can do it by adding the following property to your YAML file:
+
+```yaml
+bootstrap:
+  preinstallCheck:
+    enabled: false
+```
+
+If the pre-install checks fail, for example, if Kafka or PostgreSQL are not available, you can retrieve the logs for a
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md:687]:
```suggestion
If the pre-install checks fail (for example, if Kafka or PostgreSQL are not available), you can retrieve the logs for a
```

File: content/en/platform/corda/5.1/reference/corda-cli/preinstall.md [hunk 5]
```diff
@@ -11,7 +11,11 @@ title: "preinstall"
 ---
 # preinstall
 This section lists the {{< tooltip >}}Corda CLI{{< /tooltip >}} `preinstall` arguments. You can use these commands to manually
-check Corda's configuration.
+check Corda's configuration and to confirm that all the Corda deployment [prerequisites]({{< relref "../../deploying-operating/deployment/prerequisites.md" >}})
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.1/reference/corda-cli/preinstall.md:14]:
```suggestion
check Corda's configuration and to confirm that all the Corda deployment [prerequisites]({{< relref "../../deploying-operating/deployment/prerequisites.md" >}})
```

Should it be "deployment.... _is_", or "deployments.... _are_"?

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md (hunk 4 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md:687]: leaving as is

File: content/en/platform/corda/5.1/reference/corda-cli/preinstall.md (hunk 5 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/5.1/reference/corda-cli/preinstall.md:14]: "... to confirm that all the Corda deployment [prerequisites] _are_ running..."

"are" refers to prerequisites

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**